tests/api.c

```diff
@@ -14840,6 +14841,56 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
 #endif
 } /* END test_wc_PKCS7_EncodeEncryptedData() */
 
+/*----------------------------------------------------------------------------*
+ | hash.h Tests
+ *----------------------------------------------------------------------------*/
+static void test_wc_HashInit(void) 
+{
+    /*enum for holding supported algorithms, #ifndef's restrict if disabled*/
+    enum wc_HashType enumArray[] = {
+    #ifndef NO_MD5
+            WC_HASH_TYPE_MD5,
+    #endif
+    #ifndef NO_SHA
+            WC_HASH_TYPE_SHA,
+    #endif
+    #ifndef WOLFSSL_SHA224
+            WC_HASH_TYPE_SHA224,
+    #endif 
+    #ifndef NO_SHA256  
+            WC_HASH_TYPE_SHA256,      
```

Comment:
Please remove trailing whitespace on the lines around this enumArray, thanks.

Reply:
Trailing whitespace / blank spaces are still present on lines in this enum, please fix.

Reply:
thanks